docs/user/user/metrics/rate.md

```diff
@@ -0,0 +1,129 @@
+# Rate
+
+Used to count how often something happens relative to how often something else happens.
+Like how many documents use a particular CSS Property,
+or how many HTTP connections had an error.
+You can think of it like a fraction, with a numerator and a denominator.
+
+All rates start without a value.
+A rate with a numerator of 0 is valid and will be sent to ensure we capture the
+"no errors happened" or "no use counted" cases.
+
+> **IMPORTANT:** When using a rate metric, it is important to let the Glean metric do the counting.
+  Using your own variable for counting and setting the metric yourself could be problematic:
+  ping scheduling will make it difficult to ensure the metric is at the correct value at the correct time.
+  Instead, count to the numerator and denominator as you go.
+
+## Configuration
+
+Say you're adding a new rate for how often HTTP connections have errors.
+First you need to add an entry for the rate to the `metrics.yaml` file:
+
+```YAML
+network:
+  http_connection_error:
+    type: rate
+    description: >
+      How many HTTP connections error out out of the total connections made.
+    ...
+```
+
+### External Denominators
+
+If several rates share the same denominator
+(from our example above, maybe there are multiple rates per total connections made)
+then the denominator should be defined as a `counter` and shared between
+`rates` using the `denominator_metric` property:
+
+```YAML
+network:
+  http_connections:
+    type: counter
+    description: >
+      Total number of http connections made.
+    ...
+
+  http_connection_error:
+    type: rate
+    description: >
+      How many HTTP connections error out out of the total connections made.
+    denominator_metric: network.http_connections
+    ...
```

Comment:
*Maybe* we add a second numerator just to drive home the point about sharing the denominator?